crates/bitwarden-uniffi/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use bitwarden::client::client_settings::ClientSettings;
+
+uniffi::include_scaffolding!("sdk");
+
+pub struct Client(bitwarden::Client);
+
+impl Client {
+    pub fn new(_settings_input: String) -> Self {
+        let settings = Self::parse_settings(None);
+        Self(bitwarden::Client::new(settings))
+    }
+
+    pub fn run_command(&self, input_str: String) -> String {
+        input_str
+    }
+
+    fn parse_settings(_settings_input: Option<String>) -> Option<ClientSettings> {
+        None
+    }
```

Comment:
This is precisely the json client, I don't understand why we need a new struct to represent a uniffi client

Reply:
Yes it is. There is a follow up PR which changes that though. I wanted to keep this PR somewhat light and not introduce more work into it, it's essentially the minimum code we needed to get something working, you might notice that `run_command` doesn't actually run a command just echoes the input back.